resources/init.txt

```diff
@@ -100,14 +100,14 @@ Options to control how names are displayed (if SHOW_CREATURE_NAMES is set)
 [NAMES_USE_SPECIES:YES]
 
 Enables or disables the entire on-screen-display overlay (Toggle with F2)
-[SHOW_OSD:YES]
+[SHOW_OSD:NO]
 
 Shows all creatures, for debugging. Living, dead, kidnapped, caged, EVERY single one.
 [ALLCREATURES:NO]
 
 Adds more logging information to what is written to stonesense.log. May be useful
 if trying having issues with sprite configuration tweaking.
-[VERBOSE_LOGGING:NO]
+[VERBOSE_LOGGING:YES]
```

Comment:
is this an intentional change?

Reply:
Sort of, i can remove it but i feel verbose logging should be on by default at least while we're messing with a lot of the code.

Reply:
im leaving it if there are no objections

Reply:
how verbose is verbose? this will affect players who use the plugin, not just devs

Reply:
honestly if you dont have verbose logging on it reports almost nothing. not to mention that the log file doesnt even exist unless you also turn on debug mode

Reply:
my question is more: is this useful for *players* to have enabled, or *developers*?

Reply:
fair enough. this change needs to be reworked anyways bc the file moved